diff --git a/CMakeLists.txt b/CMakeLists.txt
index 073d6c9..98e7669 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,6 +6,9 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
+# prevent empty rpath being created to comply with fedora packaging standards
+set(CMAKE_SKIP_RPATH TRUE)
+
 # Qt Stuff
 set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTORCC ON)

